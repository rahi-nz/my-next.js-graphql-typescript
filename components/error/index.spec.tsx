import React from "react";
import renderer from "react-test-renderer";
import Error from "./index";

describe("<Error/>", () => {
  let rootElement;
  let testRenderer;

  // eslint-disable-next-line prefer-const,react/display-name
  rootElement = () => <Error />;
  // eslint-disable-next-line prefer-const
  testRenderer = renderer.create(rootElement());

  it("Should render correctly", () => {
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
