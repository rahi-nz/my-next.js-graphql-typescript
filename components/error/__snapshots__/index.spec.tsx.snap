// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error/> Should render correctly 1`] = `
<div
  className="style__Container-sc-1j36exu-0 dQGOOK"
>
  <img
    alt="myApp"
    className="style__LoadingImg-sc-1j36exu-1 gZMMXC"
    src="/error.gif"
  />
  <p>
    something went wrong. Try again in a few minutes
  </p>
</div>
`;
